OrbitQt/main.cpp

```diff
@@ -237,6 +241,28 @@ int main(int argc, char* argv[]) {
 
   const auto deployment_configuration = FigureOutDeploymentConfiguration();
 
+  const auto openGlVersion = OrbitGl::DetectOpenGlVersion(&glutContext);
+
+  if (!openGlVersion) {
+    QMessageBox::critical(nullptr, QApplication::applicationName(),
+                          "OpenGL support was not found. Orbit cannot start!");
```

Comment:
I would prefer a message that provides some possible action to take. Like "Make sure you have a recent graphics driver installed and attempt again." (Or whatever would be appropriate.)

Reply:
Good idea. I guess this will only happen in rare cases on game dev machines. More likely is the remote RDP use case. So maybe something like this: "OpenGL support was not found. Please make sure you're not trying to start Orbit in a remote session and make sure you have a recent graphics driver installed. Then try again!"

Reply:
SGTM